docker/install.sh

```diff
@@ -88,6 +88,7 @@ python3.7 -m pip install lxml==4.5.0
 
 # Install CARLA
 cd /home/carma/src/
+wget "https://carla-releases.s3.eu-west-3.amazonaws.com/Linux/CARLA_0.9.10.tar.gz"
```

Comment:
When was this added? I must have missed this PR somehow, I don't think this is correct. The whole next few lines are so that you can provide your own local copy of this file instead. Otherwise we're downloading over 6GB of this package every time we build. Functionally, it's correct. So if this is what we've been testing on, I'm not going to hold this up. But we should change this back on develop or we're really slowing things down.